src/main/java/org/ow2/proactive/workflow_catalog/rest/util/ProActiveWorkflowParser.java

```diff
@@ -74,12 +74,13 @@
 
     private static final String ELEMENT_VARIABLES = "variables";
 
+    private static final String ELEMENT_TASK_FLOW = "taskFlow";
```

Comment:
The constants are ordered by lexicographic order. Please move this new one after `ELEMENT_JOB`.